exonum-java-binding/pom.xml

```diff
@@ -202,7 +202,7 @@
       <!-- An empty artifact, required while JUnit 4 is on the classpath to override its
            dependency on hamcrest.
 
-           See http://hamcrest.org/JavaHamcrest/distributables#upgrading-from-hamcrest-1x
+           See https://hamcrest.org/JavaHamcrest/distributables#upgrading-from-hamcrest-1x
```

Comment:
actually `http` works well but `https`:
![image](https://user-images.githubusercontent.com/5055052/71375873-4a645e00-25c8-11ea-9584-2b921abbf0aa.png)

Reply:
Thanks, that was committed accidentally — reverted 🙃 

I actually noticed that issue yesterday and submitted it: https://github.com/hamcrest/JavaHamcrest/issues/280